### ISSUE TITLE
[Suggestion] Add functions to screen bricks and the fix the writable screen

### ISSUE THREAD

**Issue by CoasterBlock** [tsunamayo/Starship-EVO#5203]
The screens right now are pretty much useless. It would be great if you could connect screens to the various "computer" bricks. So ship/station/kiosk etc to be able to display certain information.  Plus being able to display information about mechanical blocks such as the extention of pistons, activation of sensors and requests to docking. 

For example: 
A screen connected to a ship computer would be able to show multiple information pieces. 

- Fuel
- Heat
- Shields
- Energy
- Docking Status
- Ship size/mass information 

As for the second type of screen (the one used to make holograms);

A better UI and fix to the spacing would be nice. 
Plus an easier way to create "complex" displays. It would also be useful and fun to have ship information displayed on large free-floating holograms. 
That would be another layer to immersion, as now I have to depend on the HUD which feels very "game-y" Plus the screens feel like an obsolete/useless brick. 
The hologram screen also deleted text when pressing "enter".

**Comment by Dwarf-LordPangolin** [comment]
I second this. I completely understand if this is a lower priority issue, but the screens were genuinely useful for ships when they were able to display information, the "hologram" effect achieved by offsetting text on the transparent screen lets people achieve some highly creative results, and the 3d ship holograms were _super_-cool.

**Comment by CoasterBlock** [comment]
@Dwarf-LordPangolin Yeah, this really isnt priority compared to bug fixes, but it was a very useful feature. The ship hologram could also be useful in that it would be able to display damage, fuel flow, or the position of certain ship components.

**Comment by TIKIRobo** [comment]
I quite like this idea and it is similar to #4040  something that ive posted about in the past

**Comment by Majikmonster** [comment]
I fourth this. Screens have been purely decorative since they lost the ability to display most system stats. Also, floating text and holograms are less visually obstructive, so they'd be used heavily by pilots and should have the ability to display ship telemetry. It would also help to have screens display info from computer cores so you don't have to drop everything to interact with the turret/ship/station/hovercraft/mech computer UI to check their values every time something changes.

**Comment by Reason147** [comment]
Having screens display video feed for larger ships of the outside would use the existing cameras while sitting on your bridge.

**Comment by tsunamayo** [comment]
Yes, the problem is that I need a complete refactoring on the gfx side as right now it is one draw call per screen, which is not acceptable (as you guys like to put 100s of those on a ship), so that would be shooting me on the foot. It is planned but lower priority unfortunately (for after survival...). Thanks
